src/core.js

```diff
@@ -995,32 +995,38 @@ define([
      *
      * NOTE this is done once when Vellum is loaded. These handlers must work
      * for multiple Vellum instances on the same page.
+     *
+     * These handlers need to be executed AFTER jstree's own dnd_move and
+     * dnd_stop handlers, so wait to attach them until the first time a
+     * drag begins.
      */
-    $(document).on("dnd_move.vakata.jstree", function (e, data) {
-        var source = $(data.data.obj),
-            target = $(data.event.target),
-            inst = $.jstree.reference(target);
-        if (!inst && target.vellum("get") === source.vellum("get")) {
-            // only when not dragging inside the tree
-            if (target.closest('.jstree-drop').length) {
-                data.helper.find('.jstree-icon').removeClass('jstree-er').addClass('jstree-ok');
-            } else {
-                data.helper.find('.jstree-icon').removeClass('jstree-ok').addClass('jstree-er');
+    $(document).one("dnd_move.vakata.jstree", function (e, data) {
```

Comment:
I don't think so - at the point `ready` is triggered, you might be certain that the code that attaches the document ready handler containing the jstree dnd_move event has executed, but I don't think you're guaranteed the document itself is ready, so you aren't certain that ready handler has executed and attached the dnd_move handler.